admin/monitoring/alerts.md

```diff
@@ -0,0 +1,172 @@
+# Custom alerting for your community
+
+Grafana can be used to create *custom alerts* that are specific to your
+community to fit into specific workflows you may have. [Grafana Alerting](https://grafana.com/docs/grafana/latest/alerting/)
+is very powerful, and upstream grafana has reasonably good documentation on
+its various features. This document lists some of the common things you
+can do with it, as well as some popular alerts our communities have found
+helpful.
+
+## Why?
+
+As admins of a JupyterHub, you may want to be proactively notified when
+certain conditions are met so you may intervene with your users. For example,
+you may want to know when users are nearing their home directory limit so
+you can reach out to them to ask if they need help or to encourage them to
+clean up their home directory.
+
+```{note}
+2i2c has our own internal alerting system for the underlying infrastructure
+we maintain, and we will take action on those ourselves when necessary. These
+alerts are entirely distinct from that.
+```
+
+## Where can alerts be sent?
+
+Grafana supports sending alerts to [many kinds of *contact points*](https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/#supported-contact-point-integrations). In this document,
+we will use Slack as the example as many of our communities use it. You should
+be able to follow upstream Grafana documentation to set up alerting to (almost)
+any kind of contact point you may already be using and that fits within your
+existing workflows.
+
+```{note}
+The one exception to this is the "Email" contact point. Sending email reliably
```

Comment:
I wonder if it's worth noting that even though Email is listed in the contact points dropdown, it won't work for 2i2c Grafana instances?